src/h5web/providers/mock/values.ts

```diff
@@ -22,6 +22,7 @@ export const mockValues = {
   raw_large: { str: '.'.repeat(1000) },
   scalar_int: 0,
   scalar_str: 'foo',
+  scalar_bool: true,
```

Comment:
`'true'` should be a string to be consistent with our providers

Reply:
Actually, I checked once again and data requests return booleans :roll_eyes: